crates/api_common/README.md

```diff
@@ -24,3 +24,10 @@ As you can see, each API endpoint needs a parameter type ( GetPosts), path (/pos
 For a real example of a Lemmy API client, look at [lemmyBB](https://github.com/LemmyNet/lemmyBB/tree/main/src/api).
 
 Lemmy also provides a websocket API. You can find the full websocket code in [this file](https://github.com/LemmyNet/lemmy/blob/main/src/api_routes_websocket.rs).
+
+## Generate TypeScript bindings
+
+TypeScript bindings (API types) can be generated by running `cargo test --features full`.
+The ts files be generated into a `bindings` folder.
```

Comment:
Output is in `crates/api_common/bindings`

Reply:
This README file is in crates/api_common. Referencing the bindings folder in it matches that path.